src/pages/admin/admin-newsfeed/admin-newsfeed.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card } from "components/card/card";
+import { Checkbox } from "components/form-components/checkbox/checkbox";
+import { Input } from "components/form-components/input/input";
+import { Grid } from "components/grid/grid";
+import { useState } from "react";
+import { FormProvider } from "react-hook-form";
+
+export const AdminNewsfeed = () => {
+  const [title, setTitle] = useState("");
+  const [text, setText] = useState("");
+  
+  return (
+    <div className="flex items-center justify-center ">
+      <div className="my-profile-page-container-without-grid-template ">
+        <div className="my-profile-side-bar debug">
+          <div className="grid grid-rows-12">
+            <form className="px-10">
+              <br/>
+              <label className="px-2">Title</label>
+              <input
+                name="title"
+                value={title}
+                onChange={(e) => {
+                  setTitle(e.target.value);
+                }}
+              />
+              <br/>
+              <br/>
+              <label className="px-2">Text</label>
+              <input
```

Comment:
Give the "text" input a size of 25vh or 50vh so that it looks like it fits more text in it.

Reply:
don't vh is viewport height, that a quarter of the height of the screen